app/views/partials/_cover_search.html.haml

```diff
@@ -1,4 +1,4 @@
 .cover
   .search-bar
     %i.icon-search
-    %input#search-bar-text{placeholder: "search something", type: "text", value: ""}
+    -#%input#search-bar-text{placeholder: "search something", type: "text", value: ""}
```

Comment:
Problem is, I don't know yet if it's gonna be necessary. I'll update the file in question after tomorrow's meeting with the client.